packages/a/ag-grid.json

```diff
@@ -4,36 +4,44 @@
   "description": "Advanced Framework Agnostic Javascript Datagrid.",
   "repository": {
     "type": "git",
-    "url": "https://github.com/ceolter/ag-grid.git"
+    "url": "https://github.com/ag-grid/ag-grid"
   },
   "keywords": [
-    "web-components",
+    "ag",
+    "ag-grid",
+    "datagrid",
+    "data-grid",
+    "datatable",
+    "data-table",
     "grid",
-    "data",
     "table",
-    "angular"
+    "react",
+    "table",
+    "angular",
+    "angular-component",
+    "react",
+    "react-component",
+    "angularjs",
+    "reactjs"
   ],
   "license": "MIT",
   "homepage": "http://www.ag-grid.com/",
   "autoupdate": {
     "source": "git",
-    "target": "git://github.com/ceolter/ag-grid.git",
+    "target": "git://github.com/ag-grid/ag-grid.git",
     "fileMap": [
       {
         "basePath": "dist",
```

Comment:
I'm guessing this might need to become `community-modules/all-modules/dist`, based on https://github.com/ag-grid/ag-grid/tree/v25.3.0/community-modules/all-modules/dist

Reply:
Thanks @MattIPv4 - are you saying the "basePath" needs to change? Can I ask where cdnjs looks for these files? I ask as we use release branches (and tag) them, so we don't use master/main for this, or the default branch (latest). 

Apologies if this is documented somewhere - I did look but was unable to find anything on how to configure the json file

thanks!

Reply:
👋 Git-based auto-update is done from git tags -- I've linked to your lastest tag above, and what I believe the new `basePath` needs to be to get the new files that should be distributed, unless those aren't what should be distributed?

Reply:
Thanks, I've updated it with what the path should be (you were close :-) ) - thanks for your help